include/glow/Optimizer/Optimizer.h

```diff
@@ -75,9 +75,9 @@ Function *profileQuantization(PlaceholderBindings &bindings, Function *F,
 std::unique_ptr<IRFunction> generateAndOptimizeIR(Function *F, const Backend &B,
                                                   bool shouldShareBuffers);
 
-/// Optimize the Function \p F given compilation options \p opts for Backend \B.
-void optimizeFunction(Function *F, const Backend &B,
-                      const CompilationOptions &opts);
+/// Optimize the Function \p F given compilation context \p cctx for Backend \B.
```

Comment:
//  \returns ... 

It is not clear from the signature if the function is cloned and optimized or just modified in place.